qiskit/synthesis/evolution/suzuki_trotter.py

```diff
@@ -105,51 +108,91 @@ def __init__(
             ValueError: If order is not even
         """
 
-        if order % 2 == 1:
+        if order > 1 and order % 2 == 1:
             raise ValueError(
                 "Suzuki product formulae are symmetric and therefore only defined "
-                "for even orders."
+                f"for when the order is 1 or even, not {order}."
             )
         super().__init__(order, reps, insert_barriers, cx_structure, atomic_evolution, wrap)
 
-    def synthesize(self, evolution):
```

Comment:
this is not deprecated - right, just called from ProductFormula.synthesize?

Reply:
Correct 🙂